### DISCUSSION TITLE
Bypassing Cloudflare not working anymore

### DISCUSSION THREAD

**Discussion by qomhmd** [seleniumbase/SeleniumBase#1952] [Q&A] [answered]
Provided by @mdmintz at [Stackoverflow](https://stackoverflow.com/a/76567468/5251465), the solution does not work for me anymore since yesterday. The page just refreshes for a few tries.

```python
from seleniumbase import SB

def verify_success(sb):
    sb.assert_text("some text to be detected so to probably determine as success", timeout=7.25)
    sb.post_message("Selenium wasn't detected!", duration=2.8)
    sb._print("\n Success! Website did not detect Selenium! ")

with SB(uc=True, incognito=True) as sb:
    sb.open(some_url)
    try:
        verify_success(sb)
    except Exception:
        if sb.is_element_visible('input[value*="Verify"]'):
            sb.click('input[value*="Verify"]')
        elif sb.is_element_visible('iframe[title*="challenge"]'):
            sb.switch_to_frame('iframe[title*="challenge"]')
            sb.click("span.mark")
        else:
            raise Exception("Detected!")
        try:
            verify_success(sb)
        except Exception:
            raise Exception("Detected!")
```

**Comment by mdmintz** [comment, marked as answer]
This was resolved by ``4.15.14`` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.15.14

**Reply by qomhmd** [reply]
Either it doesn't work for me after updating (v4.15.15)

**Reply by mdmintz** [reply]
CI is green, https://github.com/mdmintz/undetected-testing/actions/runs/5551593733, and tests are also working locally for me. Make sure to add ``--uc`` as a ``pytest`` command-line option when running with ``pytest``. Try https://github.com/seleniumbase/SeleniumBase/blob/master/examples/uc_cdp_events.py and https://github.com/seleniumbase/SeleniumBase/blob/master/examples/verify_undetected.py
And note that you may need to add ``--devtools`` for better evasion with ``--uc``.
With the ``SB()`` format, use: ``SB(uc=True, devtools=True)``.

**Reply by qomhmd** [reply]
Thanks, I will. FYI, I've tried combinations of devtools, uc, incognito, and test. The URL is https://ir-appointment.visametric.com/de if it is shown for your IP range.
